how-to-guides/create-delete-projects/scripts/create_delete_projects.py

```diff
@@ -1,18 +1,17 @@
 from neptune import management
 
-PROJECT_NAME= "<WORKSPACE/PROJECT>"
-API_TOKEN= "<YOUR_API_TOKEN>"
+PROJECT_NAME = "YOUR_WORKSPACE/YOUR_PROJECT"
+API_TOKEN = "YOUR_API_TOKEN"
+KEY = "YOUR_PROJECT_KEY"
 
 # create project and choose whether the project should be public or private
-management.create_project(name=PROJECT_NAME, 
-                          api_token=API_TOKEN, 
-                          key="AMA", 
-                          visibility="pub")
+management.create_project(
+    name=PROJECT_NAME, api_token=API_TOKEN, key=KEY, visibility="pub"
+)
 
 # list projects that you have access to
```

Comment:
here inther print all projects, or mention in the comment that last 10 projects will be returned